### PR TITLE
ViewTrips Page Styling: Font Awesome Setup

### PR DESCRIPTION
**Pull request by zghera**
### What is a quick description of the change?

Add capability to use [Font Awesome](https://fontawesome.com/start) in the project.

### Is this fixing an issue?

none

### Are there more details that are relevant?

Changes more specifically:
- `package-lock.json` and `package.json`, : Installation
- `IconLibrary/index.js`: File that defines all icons used in the project. This is the recommended way to import icons (more details on why this is used [here](https://github.com/FortAwesome/react-fontawesome#build-a-library-to-reference-icons-throughout-your-app-more-conveniently)).
- `App/index.js`: Import `IconLibrary/index.js` so any icon placed in the library can be used anywhere in the project.
- `trip.js`: Example use of the `FontAwesomeIcon` component with the one icon added to the icon library.

### Check lists (check `x` in `[ ]` of list items)

- [  ] Test written/updating
- [  ] Tests passing
- [x] Coding style (indentation, etc)

None: styling.

### Any additional comments?

Picture of the example icon:
image

### CODE REVIEW

**Comment by ananyeet** [comment]
The link in the PR seems to be an older version of fontAwesome, it looks like [version 5](https://fontawesome.com/start) is out.

**Comment by zghera** [comment]
> The link in the PR seems to be an older version of fontAwesome, it looks like [version 5](https://fontawesome.com/start) is out.

Just updated. Thanks for the catch!

File: frontend/src/components/ViewTrips/trip.js [hunk 1]
```diff
@@ -2,8 +2,9 @@ import React, {useState, useEffect} from 'react';
 
 import { Accordion, Button, Card } from 'react-bootstrap';
 
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import authUtils from '../AuthUtils';
-import { timestampToISOString, getDateRangeString } from '../Utils/time.js';
+import { getDateRangeString, timestampToISOString } from "../Utils/time.js";
```
Comment (Hiram-Silvey) [inline frontend/src/components/ViewTrips/trip.js:7]: Why is this changed?

Comment (zghera) [reply frontend/src/components/ViewTrips/trip.js]: Hmm I don't remember doing that so that must have happened during the crazy merge conflict resolutions before the presentation. Fixed.
